data/json/npcs/talk_tags.json

```diff
@@ -565,7 +583,122 @@
           "Look out!  A",
           "<swear!>, here comes a <swear>"
         ]
-    },{
+    },
+    {
+      "type": "snippet",
+      "category": "<general_danger>",
+      "text": [
+        "Look sharp!  Things are heating up.",
+        "Hey, <name_g>!  Hostiles inbound.",
+        "Uh, <name_g>?  Are we fighting or leaving?",
+        "Naptime is over.",
+        "Who's there?",
+        "Hello?",
+        "Look alive!"
```

Comment:
You could probably get away with adding <general_danger_h> to this list so that friendly NPCs can still occasionally toss out some colourful language.